src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -186,6 +186,7 @@ public function handlePayoutPaid(Event $event): Response
     public function handleChargeRefunded(Event $event): Response
     {
         $chargeId = $event->data->object->id;
+        $amountRefunded = (float) $event->data->object->amount_refunded;
```

Comment:
@tbg-jasonfung wouldn't it make more sense to cast the other value to int, since both are whole numbers (pence)? Alternatively we could use `bccmp()` so we're not concerned about what type the numbers started as.